config/config.yaml

```diff
@@ -9,10 +9,10 @@ gitlab_container_pass: {{ environment_variable.GITLAB_CONTAINER_PASS | default('
 gitlab_container_user: {{ environment_variable.GITLAB_CONTAINER_USER | default('_WILL_NOT_WORK_') }}
 # DO NOT store the token in git.
 gitlab_reg_token: {{ environment_variable.GITLAB_REG_TOKEN | default('_NEVER_GONNA_GET_IT_') }}
-gitlab_runner_instance_type_prod: 'r5a.2xlarge'
+gitlab_runner_instance_type_prod: 'r5a.4xlarge'
```

Comment:
why not M7g or R7g?

Reply:
We are needing memory optimized. The `r7g.4xlarge` is a little bit less expensive than the`r5a.4xlarge` (but only $.05/hr). Though, it is ARM based and makes me a little concerned about software compatibility.
I would still want to go with the r5a.

Reply:
+1 on ARM, have the same concern.